File: PooCC01CNetz/Semana3/Java/Ejercicios1/src/Alumno.java
public class Alumno {
    private String nombre;
    private int matricula;

    public Alumno(String nombre, int matricula) {
        this.nombre = nombre;
        this.matricula = matricula;
    }
    
    public Alumno() {}

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setMatricula(int matricula) {
        this.matricula = matricula;
    }

    public int getMatricula() {
        return matricula;
    }
}

File: PooCC01CNetz/Semana03/Python/Ejercicios1/Alumno.py
class Alumno:
    def __init__(self, nombre = "", matricula = 1):
        self.nombre = nombre
        self.matricula = matricula

    def setNombre(self, nombre):
        self.nombre = nombre
    
    def getNombre(self):
        return self.nombre
    
    def setMatricula(self, matricula):
        self.matricula = matricula

    def getMatricula(self):
        return self.matricula